<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_about</class>
 <widget class="QWidget" name="widget_about">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>About Ninja Forge</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0" colspan="3">
    <widget class="QFrame" name="frame_main">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>210</width>
       <height>110</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>590</width>
       <height>390</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QPlainTextEdit" name="editbox_about">
        <property name="plainText">
         <string/>
        </property>
        <property name="placeholderText">
         <string>*about goes here*</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="button_close">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Close This Dialog</string>
        </property>
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="text">
         <string>&amp;Close</string>
        </property>
        <property name="icon">
         <iconset theme="window-close">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2" alignment="Qt::AlignLeft|Qt::AlignVCenter">
    <widget class="QLabel" name="label_version">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Version</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" alignment="Qt::AlignLeft|Qt::AlignVCenter">
    <widget class="QLabel" name="label_name">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_close</sender>
   <signal>clicked()</signal>
   <receiver>widget_about</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>542</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
